### PR TITLE
[feature1] First feature

### PR DESCRIPTION
**Pull request by alexgithubber09**
The following changes were made.

- create 2 files.

### CODE REVIEW

File: new-feature.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is new feature in the feature1
```
Comment (AlexandrRevenko) [inline new-feature.txt:1]: good job!

Comment (AlexandrRevenko) [reply new-feature.txt:1]: Thanks for comment.

Comment (alexgithubber09) [reply new-feature.txt:1]: thanks for comment

**Review by AlexandrRevenko — APPROVED** [review]
Approve

**Pull request merged**